test/spec/bowlingGameSpec.js

```diff
@@ -0,0 +1,65 @@
+describe('bowlingGame', function() {
+
+  beforeEach(function() {
+      bowlingGame = new BowlingGame;
```

Comment:
Missing '()' invoking a constructor.<br>'bowlingGame' is not defined.<br>'BowlingGame' is not defined.